docs/authoring-json-rules.md

```diff
@@ -0,0 +1,46 @@
+# Authoring Rules
+
+## Overview
+Rules are authored in JSON. Every rule contains metadata (`name`, `description`, `type`) and evaluation data. All rules can executed against a target that can be generated from the CLI input.  
+
+``` json
+  {
+    "name": "rule name",
+    "description": "short description of the rule",
+    "type": "type of rule",
+  },
+```
+*not yet implemented*: 
```

Comment:
nit: In general, we should probably leave unimplemented features out of docs - there's always a possibility we just never come back & do the thing, or that it gets implemented differently